design/sequence/finish-sd.puml

```diff
@@ -0,0 +1,30 @@
+@startuml
```

Comment:
This sequence diagram does not reflect what we discussed on Monday.
The pipeline should be ServerCommandHandler -> Database -> Scheduler -> Dispatcher.